code/game/gamemodes/changeling/traitor_chan.dm

```diff
@@ -43,6 +43,7 @@
 		for(var/j = 0, j < num_changelings, j++)
 			if(!possible_changelings.len) break
 			var/datum/mind/changeling = pick(possible_changelings)
+			antag_candidates -= changeling

```

Comment:
God fucking damn your code doesn't even work after all this yelling I was about to merge it to be done with it